### PR TITLE
add parachains section, add launch guide

### PR DESCRIPTION
**Pull request by NukeManDan**
This introduces a new section for Parachains and the launch guide based on this: https://hackmd.io/YE6fP-hRSy-g9GC6xkxzng

It is the first step at getting more parachain content on the devhub (#1027)

Open to comment on what else should be added/edited here 🙏🏼

### CODE REVIEW

**Review by jimmychu0807 — APPROVED** [review]
@NukeManDan 

Thank you for this comprehensive notes! I have done quite some editing to this document. It originally read like some kind of internal bullet-point notes that we write for ourselves.

@sacha-l I really want to have you go thru this doc. I foresee this doc is quite important for teams adding parachain functionality, and so worth spending some time reviewing this PR soon. Thanks.

File: docs/knowledgebase/parachains/launch.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+> **NOTE**: Make sure to deploy the _compressed_ (not compact) version of your runtime to avoid
+> complications.
+
+- We advise you to launch your parachain being as slim as possible and do runtime upgrades to
+include functionality. The reason behind this is that when performing a runtime upgrade both the
+previous runtime and the new runtime are included in the PoV and therefore if the changes are too
+big it might not go through. **Always use the compressed version of the runtime to lower the amount
+of data being transferred.**
+
+  Still, in order to avoid storage upgrades, you could do the following:
```
Comment (riusricardo) [inline docs/knowledgebase/parachains/launch.md:73]: Is this going to be the default recommendation? This has been told to a few teams for now but is a hack and I don't think that is a good idea to make it a standard. 
Maybe @bkchr can give his PoV here.

File: docs/knowledgebase/parachains/launch.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+- A newly compressed Wasm generated for the runtime and set on-chain (and don't forget to [increment your `spec_version`](../runtime/upgrades#runtime-versioning)).
+
+- State migrations from the old runtime to the new runtime implemented and executed in this Wasm
+(for more information refer to [Substrate Runtime Migration Guide](https://hackmd.io/BQt-gvEdT66Kbw0j5ySlWw)
+and [this repository](https://github.com/apopiak/substrate-migrations)).
+
+#### Standard Protocol
+
+> For a demo of solo-chain upgrades, see [this tutorial](../../tutorials/forkless-upgrade/).
```
Comment (riusricardo) [inline docs/knowledgebase/parachains/launch.md:105]:
```suggestion
> For a demo of runtime upgrades, see [this tutorial](../../tutorials/forkless-upgrade/).
```

**Review by riusricardo — CHANGES_REQUESTED** [review]
This docs shouldn't be in the Substrate DevHub. I suggest to move them to polkadot wiki docs.

**Comment by NukeManDan** [comment]
closing in favor of a rework and splitting between the wiki and here

**Review by sacha-l — COMMENTED** [review]
Left a few comments and suggestions. Saw that this got closed while I was doing my review so I sort of rushed through the end. Thought best to wait until post-refactor. Great initiative. Whether this lives on Substrate's devhub in some form, or Polkadot's wiki, this is super useful and needed.

**Comment by NukeManDan** [comment]
thanks for the review @sacha-l - added in your edits.

**Pull request closed without merging**